src/js/helpers/covid19Helper.js

```diff
@@ -6,11 +6,13 @@
 import { useState } from 'react';
 import { snakeCase } from 'lodash';
```

Comment:
we are importing lodash twice. just use this statement here `import { snakeCase, min, max } from 'lodash';`